src/JitInterface/src/CorInfoImpl.cs

```diff
@@ -2691,9 +2698,146 @@ private void allocUnwindInfo(byte* pHotCode, byte* pColdCode, uint startOffset,
                 blobData[i] = pUnwindBlock[i];
             }
 
+            var target = _compilation.TypeSystemContext.Target;
+
+            if (target.Architecture == TargetArchitecture.ARM64 && target.OperatingSystem == TargetOS.Linux)
+            {
+                blobData = CompressARM64CFI(blobData);
```

Comment:
It may be useful to add a short comment about what this does and why RyuJIT cannot just emit the right CFI opcodes in the first place.

Is this compression just an optimization; or is it require to make thing work? Would it make sense to enable it on other platforms too?

Reply:
When I implemented it I made sure that the CFI data is stored i the same way the C++ compiler does it as just using the unmodified data didn't worked. So maybe not everything is needed.
I did it here and not in RyuJIt just to minimize the changes in RyuJIT itself as if I understand it correctly beside of CoreRT the CFI data are never needed.

Reply:
Thanks for the context. It would be nice to capture it in a comment.